### PR TITLE
Do not allow creation of groups with empty names

### PR DESCRIPTION
**Pull request by nilsding**

### CODE REVIEW

**Comment by nilsding** [comment]
Nice.

**Comment by pixeldesu** [comment]
Nice.

**Review by pixeldesu — APPROVED** [review]

File: spec/models/group_spec.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    end
+
+    context "when display name is '1' (valid)" do
+      let(:display_name) { '1' }
+
+      it { is_expected.to be true }
+    end
+
+    context 'when display name is the letter E 621 times (invalid, too long)' do
+      let(:display_name) { 'E' * 621 }
```
Comment (raccube) [inline spec/models/group_spec.rb:49]: ugh.

**Review by raccube — APPROVED** [review]

**Comment by raccube** [comment]
Nice.

**Pull request merged**